medical_prescription_state_verify/models/medical_prescription_order.py

```diff
@@ -18,29 +18,16 @@ class MedicalPrescriptionOrder(models.Model):
             that has been defined in `_ALLOWED_CHANGE_STATES`)
         _ALLOWED_CHANGE_STATES: `list` of keys defining status types in which
             an RX can be moved to after being verified
+            4 - Verified
```

Comment:
These are the stage_ids, which seem to be set by the sequence of stages in the data files. I wanted to define their states by their name, but I was having difficulty getting the name from the stage_id. Are the ids OK, or should I try to use the names?